thirdparty/runtime_environments.go

```diff
@@ -204,3 +205,75 @@ func (c *RuntimeEnvironmentsClient) getImageDiff(ctx context.Context, opts Image
 	}
 	return filteredChanges, nil
 }
+
+// ImageHistoryInfo represents information about an image with its AMI and creation date.
+type ImageHistoryInfo struct {
+	AMI          string
+	CreationDate time.Time
+}
```

Comment:
Ahh I see what issue you ran into. For now, why don't we just parse this stuff as a regular string and then see if it makes sense to handle the timestamp conversion in the `getEvents` method 🤔

To tell the `gimlet.GetJSON` what the expected json tags are, you can do this:
```suggestion
type ImageHistoryInfo struct {
	AMI          string `json:"ami_id"`
	CreationDate string `json:"created_date"`
}
```